go.mod

```diff
@@ -8,9 +8,12 @@ require (
 	github.com/BurntSushi/graphics-go v0.0.0-20160129215708-b43f31a4a966 // indirect
 	github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802
 	github.com/BurntSushi/xgbutil v0.0.0-20160919175755-f7c97cef3b4e
+	github.com/fzipp/gocyclo v0.0.0-20150627053110-6acd4345c835 // indirect
```

Comment:
Oh, sorry, I think these lines have been added by you writing commands in ther terminal and are not really project dependencies...